sensors/sensors.go

```diff
@@ -72,16 +72,17 @@ func NewLidar(
 		return Lidar{}, errors.Wrapf(err, "error getting lidar camera %v for slam service", cameraName)
 	}
 
-	// If there is a camera provided in the 'camera' field, we enforce that it supports PCD.
-	properties, err := newLidar.Properties(ctx)
-	if err != nil {
-		return Lidar{}, errors.Wrapf(err, "error getting lidar camera properties %v for slam service", cameraName)
-	}
+	// rplidar does not yet support properties, see https://viam.atlassian.net/browse/RSDK-4600
```

Comment:
hot fix is in, you should be able to comment back in this code

Reply:
Yes and no - I'm having issues installing rplidar module from source

Reply:
Spoke about offline